Makefile

```diff
@@ -9,9 +9,8 @@ else
 endif
 
 GO_PKG=github.com/prometheus-operator/prometheus-operator
-REPO?=quay.io/prometheus-operator/prometheus-operator
-REPO_PROMETHEUS_CONFIG_RELOADER?=quay.io/prometheus-operator/prometheus-config-reloader
-REPO_PROMETHEUS_OPERATOR_LINT?=quay.io/prometheus-operator/prometheus-operator-lint
```

Comment:
Lint image is not used anywhere :shrug: 

For other images, I changed the variable names for clarity.

Reply:
should we remove `po-lint` completely?

Reply:
I do believe there are better tools for the job (like `kubeconform`) and we should point to those instead of maintaining our own tool for the same thing. Plus the same feature is actively being implemented in `kubectl`.

However, let's do that in a separate PR (here is an issue: https://github.com/prometheus-operator/prometheus-operator/issues/4099). I'll leave this line here though as it is dead code either way.